### ISSUE TITLE
Assembler: Copy review for patterns and styles screens

### ISSUE THREAD

**Issue by taipeicoder** [Automattic/wp-calypso#81307]
## Description

Related to p1693490882523839-slack-CRWCHQGUB. This issue tracks the discussion to improve the copy for the patterns and styles screen in the Assembler. See the current copy below:

| Patterns screen | Styles screen |
| --- | --- |
| ![Image](https://user-images.githubusercontent.com/797888/264868974-f0c57a23-0cb5-4685-a0bd-2c90e7642885.png)|![Image](https://user-images.githubusercontent.com/797888/264869070-a3765933-e996-43e4-8e00-f82978fe0087.png)|

Feedback from the thread linked above:
- The back button should simply say "Back to patterns" or "Back to styles", instead of "Back to Design your own" or "Back to Pick your style" (not shown in the screenshots above).
- Proposed copy for the subheading text:

| Option | Patterns screen | Styles screen |
| --- | --- | --- |
| A | Use our library of patterns to build your homepage. |  Choose our expertly curated color palettes and font parings. |
| B | Design your homepage from our library of patterns. | Add style to your page with our expertly curated color palettes and font parings.|

In short, we're looking to review the copy in the:
1. Back button.
2. Heading text.
3. Subheading text.

## Acceptance Criteria
- [ ] Finalize on the new copy with @michaelpick and Francisco.
- [ ] Update the new copy.

**Comment by taipeicoder** [comment]
Continuing the discussion in this thread p1693553440960149-slack-C048CUFRGFQ. Will update this issue when things are finalized.

**Comment by autumnfjeld** [comment]
FYI: here is the copy doc with ideas https://docs.google.com/document/d/1-SffV2neFGMUp_NqzDmbQMib13d5_kpnNZPYTXe9SgE/edit#heading=h.jjimrjl4weib

**Comment by autumnfjeld** [comment]
@taipeicoder 
My suggestions.   I'm assuming we want a short one-line heading, so the copy I choose is to achieve that.  Following from this discussion with @lucasmendes-design 

## PATTERNS PANEL:


CURRENT:
<img width="345" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5665959/3d8242dd-abf7-4d7c-8333-aa4165beb84d">


NEW:
**Heading:** Select your patterns
**Subheading:**  Create your homepage from our library of patterns.


----------------------------

## STYLES PANEL

CURRENT
<img width="331" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5665959/8a237729-bed6-48a9-b3dc-bc84fd8ee0a1">


NEW
**Heading:** Pick your styles
**Subheading:**  Add style to your page with our expertly curated color palettes and font parings.

**Comment by autumnfjeld** [comment]
@lucasmendes-design  Can we move forward with the copy I suggested?

**Comment by arthur791004** [comment]
Closing it via https://github.com/Automattic/wp-calypso/pull/81764
